src/lib/nvme_pt_ioctl.c

```diff
@@ -2271,3 +2292,27 @@ int opal_list_lr_pt(struct sed_device *dev, const struct sed_key *key,
 	opal_end_session(dev->fd, opal_dev);
 	return ret;
 }
+
+int opal_write_shadow_mbr_pt(struct sed_device *dev, const struct sed_key *key,
+			const uint8_t *from, uint32_t size, uint32_t offset)
+{
+	int ret;
+	struct opal_device *opal_dev;
+
+	if (from == NULL) {
+		SEDCLI_DEBUG_MSG("Must provide a valid source pointer\n");
+		return -EINVAL;
+	}
+
+	opal_dev = dev->priv;
+
+	ret = opal_start_admin1_lsp_session(dev->fd, opal_dev, key);
+	if (ret)
+		goto end_sessn;
+
+	ret = opal_write_mbr(dev->fd, opal_dev, from, offset, size);
+
+end_sessn:
+        opal_end_session(dev->fd, opal_dev);
```

Comment:
please fix coding style issues: ERROR: code indent should use tabs where possible

Reply:
will do it